src/main/java/seedu/address/ui/BookDetailsPanel.java

```diff
@@ -67,20 +68,25 @@ private void showBook(Book book) {
         });
     }
 
-    protected void scrollToTop() {
-        scrollPane.setVvalue(0);
+    public void clear() {
```

Comment:
Can this be `protected`?
Also, does this panel still properly scroll to top? Some of the code from #79 seems to be missing from here and `MainWindow`.